src/test/java/seedu/trackbeau/logic/commands/EditCustomerCommandTest.java

```diff
@@ -94,8 +95,12 @@ public void execute_someFieldsSpecifiedRemovedUnfilteredList_success() {
     public void execute_noFieldSpecifiedUnfilteredList_success() {
         EditCustomerCommand editCustomerCommand =
             new EditCustomerCommand(INDEX_FIRST_CUSTOMER, new EditCustomerCommand.EditCustomerDescriptor());
+        System.out.println(editCustomerCommand.toString());
+
         Customer editedCustomer = model.getFilteredCustomerList().get(INDEX_FIRST_CUSTOMER.getZeroBased());
 
+        System.out.println(editedCustomer.toString());
```

Comment:
I think its the same here too probably some debugging way that wasn't removed haha

Reply:
thanks for spotting, i removed it!